### PR TITLE
Mta 1619 app analysis language selection

### PR DESCRIPTION
**Pull request by HagayVider1**
See ticket: https://issues.redhat.com/browse/MTA-1619
Replaces closed PR https://github.com/windup/windup-documentation/pull/783

### CODE REVIEW

File: docs/topics/mta-web-creating-custom-migration-targets.adoc [hunk 1]
```diff
@@ -17,6 +17,8 @@ Using ready-made custom migration targets makes unnecessary the cumbersome task
 .Procedure
 
 . In *Administration* view, click *Custom migration targets*.
+. Near the top-left corner of the main pane, select if the application is *Java* or *Go*
```
Comment (anarnold97) [inline docs/topics/mta-web-creating-custom-migration-targets.adoc:20]:
```suggestion
. In the top-left corner of the main pane, select if the application is *Java* or *Go*
```

File: docs/topics/mta-web-creating-custom-migration-targets.adoc [hunk 2]
```diff
@@ -17,15 +17,27 @@ Using ready-made custom migration targets makes unnecessary the cumbersome task
...
+image::mta-web-target-language.png[Set custom migration target]
+
+. In the upper-left corner of the main pane, select the application language.
++
+[NOTE]
+====
+{ProductShortName} currently only supports Java. Go is available as a _Technology Preview_ feature.
+====
++
+. If you selected *Java*, under *Application server integration migration to* select *JBoss EAP 7* or *JBoss EAP 8*.
```
[the remark this answers is not in the document]
Comment (kpunwatk) [reply docs/topics/mta-web-creating-custom-migration-targets.adoc:31]: Screenshot from 2023-12-18 14-30-02

**Comment by kpunwatk** [comment]
Rest all look good to me

File: docs/topics/mta-web-creating-custom-migration-targets.adoc (hunk 2 above)
[the remark this answers is not in the document]
Comment (HagayVider1) [reply docs/topics/mta-web-creating-custom-migration-targets.adoc]: Done

**Review by kpunwatk — APPROVED** [review]
LGTM, Thanks !

**Review by dymurray — APPROVED** [review]
LGTM!

File: docs/topics/mta-web-creating-custom-migration-targets.adoc [hunk 3]
```diff
@@ -8,7 +8,7 @@
 
 Architects or users with `admin` permissions can create and maintain custom rulesets associated with custom migration targets. Architects can upload custom rule files and assign them to various custom migration targets. The custom migration targets are then available for selecting in the analysis configuration wizard.
 
-Using ready-made custom migration targets makes unnecessary the cumbersome task of configuring custom rules for each analysis run. This simplifies analysis configuration and execution for non-admin users or third-party developers and makes using the {ProductShortName} tool easier and more straightforward.
+Using ready-made custom migration targets makes unnecessary the cumbersome task of configuring custom rules for each analysis run. This simplifies analysis configuration and execution for non-admin users or third-party developers and makes using {ProductShortName} easier and more straightforward.
```
Comment (shawn-hurley) [inline docs/topics/mta-web-creating-custom-migration-targets.adoc:11]: nit: makes the cumbersome task of configuring custom rules for each analysis run unnecessary. 

IDK just sounds more correct to me, but I am no expert.

**Review by shawn-hurley — APPROVED** [review]
Looks good to me

**Pull request closed without merging**